plugin/todo.vim

```diff
@@ -1,4 +1,6 @@
 
-command! TodoQuickFix lua require("todo-comments.search").setqflist()
-command! TodoTelescope Telescope todo-comments todo
-command! TodoTrouble Trouble todo
+if executable('rg')
```

Comment:
I'd rather not have this here.
When a user will try to execute `TodoTrouble` etc, they will get a proper error saying they need to have ripgrep installed